sass/_footer.scss

```diff
@@ -32,10 +32,8 @@
     // As a substitute, I'm going to use the alpha-70 version of accent.
     color: var(--accent-alpha-70);
 
-    &--user {
-      margin: auto;
-      text-align: center;
-    }
```

Comment:
Is there any particular rationale for centering the copyright notice _only_ for the user defined ones?